addons/prometheusadapter/prometheusadapter.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: prometheusadapter
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.7.0-4"
-    appversion.kubeaddons.mesosphere.io/prometheusadapter: "v0.7.0"
-    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/prometheus-community/helm-charts/ef6391472c694c8cb1412b277f94ae3b7cfe972e/charts/prometheus-adapter/values.yaml"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.8.3-1"
+    appversion.kubeaddons.mesosphere.io/prometheusadapter: "v0.8.3"
+    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://github.com/prometheus-community/helm-charts/blob/61fb05e4f4ba9b6e4255a07338715414a3f493a0/charts/prometheus-adapter/values.yaml"
```

Comment:
This needs to be the raw link for the UI to consume.

Reply:
This annotation is ...something to get rid of asap.

Reply:
I updated it.

Reply:
> This annotation is ...something to get rid of asap.

I'm always open to suggestions and happy to work with UI to find consensus on a better solution.